my_game.py

```diff
@@ -68,6 +68,27 @@ def update(self):
             self.right = SCREEN_WIDTH - 1
 
 
+class Asteroid(arcade.Sprite):
+    
+    def __init__(self, center_x=0, center_y=0):
+        
+        #Initialize the asteroid
+        
+        #Graphics
+        super().__init__("images/Meteors/meteorGrey_big1.png", SPRITE_SCALING)
+        
+        self.center_x = center_x
+        self.center_y = center_y
+        self.change_x = 0
```

Comment:
0 is default for change_x and change_y. No need to specify.

Reply:
I see. Must have misunderstood yesterday :)